{{cookiecutter.package_name}}/.travis.yml

```diff
@@ -16,6 +16,7 @@ install:
   - pip install -e .
 
 script:
+  - flake8 {{cookiecutter.package_name}}
```

Comment:
:+1: for adding but I think we can eliminate the `{{cookiecutter.package_name}}` bit because we're already excluding stuff in the flake8 block in the [`setup.cfg`](https://github.com/smn/cookiecutter-python-package/blob/develop/%7B%7Bcookiecutter.package_name%7D%7D/setup.cfg#L4-L6) file.